### PR TITLE
Include dellemc.openmanage collection for Ansible 3.0

### PR DESCRIPTION
**Pull request by dmsimard**
The history of the application as well as the review is available at:
https://github.com/ansible-collections/ansible-inclusion/discussions/3

### CODE REVIEW

File: 3/collection-meta.yaml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ---
 collections:
+  dellemc.openmanage:
+    maintainers:
+      - rajeevarakkal
```
Comment (felixfontein) [inline 3/collection-meta.yaml:5]:
```suggestion
      - rajeevarakkal
    changelog-url: https://github.com/dell/dellemc-openmanage-ansible-modules/blob/collections/CHANGELOG.rst
```

File: 3/collection-meta.yaml [hunk 2]
```diff
@@ -1,5 +1,8 @@
 ---
 collections:
+  dellemc.openmanage:
+    maintainers:
```
Comment (felixfontein) [inline 3/collection-meta.yaml:4]:
```suggestion
    changelog-url: https://github.com/dell/dellemc-openmanage-ansible-modules/blob/collections/CHANGELOG.rst
    maintainers:
```

Comment (dmsimard) [reply 3/collection-meta.yaml:4]: @felixfontein I might have assumed that this wasn't required for collections that are using antsibull-changelog for relnotes given that the other changelogs in the file are markdown. Should we add it anyway ?

Comment (felixfontein) [reply 3/collection-meta.yaml:4]: They do not use antsibull-changelog, since they do not have a changelogs/changelog.yaml file.

Comment (dmsimard) [reply 3/collection-meta.yaml:4]: OH, you're right, I don't know how I missed that.

Comment (felixfontein) [reply 3/collection-meta.yaml:5]: The CHANGELOG.rst file really looks like is created by antsibull-changelog :)

**Review by felixfontein — APPROVED** [review]

**Comment by abadger** [comment]
Great!  Thank you!

**Comment by rajeevarakkal** [comment]
@felixfontein , @abadger - Thanks a lot :)

**Pull request merged**